packages/react-scripts/template/README.md

```diff
@@ -342,15 +343,15 @@ Next we add a 'lint-staged' field to the `package.json`, for example:
     // ...
   },
 + "lint-staged": {
-+   "src/**/*.{js,jsx,json,css}": [
++   "src/**/*.{js,jsx,ts,tsx,json,css,scss,html,md}": [
 +     "prettier --single-quote --write",
 +     "git add"
 +   ]
 + },
   "scripts": {
 ```
 
-Now, whenever you make a commit, Prettier will format the changed files automatically. You can also run `./node_modules/.bin/prettier --single-quote --write "src/**/*.{js,jsx}"` to format your entire project for the first time.
+Now, whenever you make a commit, Prettier will format the changed files automatically. You can also run `./node_modules/.bin/prettier --single-quote --write "src/**/*.{js,jsx,ts,tsx,json,css,scss,html,md}"` to format your entire project for the first time.
```

Comment:
If I'm not wrong `html` is not supported by prettier :)

Reply:
Prettier does have HTML support but it is currently undocumented and experimental
https://github.com/prettier/prettier/issues/5098

Reply:
I got this extensions list from [here](https://github.com/prettier/prettier/blob/1244729ad74df56dd3b607dd395f0070c5e8baa6/.pre-commit-hooks.yaml#L6), but yeah I just tested and it didn't work out of the box on an html file so I removed it from the readme. Thanks.